.github/workflows/release.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
   publish-image:
+    name: "Publish Docker Image to Docker Hub"
     runs-on: ubuntu-latest
     steps:
       -
```

Comment:
While we're touching this file, should the `name` go on the same line as `-`, thats what the other steps are doing right?